src/api/comms.js

```diff
@@ -43,7 +51,7 @@ export function checkStatus(response = {}) {
   throw error;
 }
 
-export function request(uri, options = defaultOptions) {
+export async function request(uri, options = defaultOptions) {
```

Comment:
Woah, lots are calling this function - is this definitely safe to make async?

Reply:
We have reverted it to a non-async function and the code and tests both work :)